microcosm/config/model.py

```diff
@@ -98,26 +98,44 @@ def __init__(
         self.default_factory = default_factory
         self.mock_value = mock_value
         self.required = required
-        self.nullable = nullable
-
-        # Warn when the user doesn't provide exactly one of
-        #
-        # [`required=True`, `default_value=...`]
-        #
-        # When the user is not using `default_factory`, this could be legacy
-        # code, so we just warn. In the future this will be an error, but for
-        # now we just warn in those situations which could occur as of the time
-        # this was introduced
+        self.nullable = nullable or (default_value is None)
+
+        if kwargs:
```

Comment:
We now no longer support arbitrary `kwargs`.  No idea why we supported this in the first place but it bit me when I used `default=True` instead of `default_value=True` and nothing complained

Reply:
(see globality-corp/microcosm-flask#221 for 🤦‍♂)